tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -292,6 +292,7 @@ private static void GenerateMainProject(ApiMetadata api, string directory, HashS
                 // Build-related properties
                 new XElement("Version", api.Version), // TODO: Version, or VersionPrefix/VersionSuffix?
                 new XElement("TargetFrameworks", targetFrameworks),
+                new XElement("LangVersion", "latest"),
```

Comment:
True, although I believe the language team is *more* likely to make "breaking in corner cases" changes within minor versions if they're changing a new feature that hasn't been used as much yet. (It's not really semver, basically.)

Let's stick with latest :)

LGTM.